site/content/docs/5.2/content/tables.md

```diff
@@ -110,6 +150,7 @@ Highlight a table row or cell by adding a `.table-active` class.
 
 ```html
 <table class="table">
+  <caption class="visually-hidden">Boosted table with an active row and cell</caption>
```

Comment:
For those use cases, we want in the documentation to show the feature so IMO we can drop in the example part the caption; what's interesting here is the `.table-*` class. If a modification is done, should be done everywhere that is linked to this concept.